3711/1/scanner/yaml.rb

```diff
@@ -0,0 +1,34 @@
+require 'yaml'
+
+module Scanner
+  class Yaml
+    def initialize(args)
+      @path = args['file'] || 'rubygems.yml'
```

Comment:
I'm not sure if if it's good idea to use default file instead of letting user know that he didn't configure argument in the right way